lib/flipper/api/v1/decorators/feature.rb

```diff
@@ -0,0 +1,33 @@
+require 'delegate'
+require 'flipper/api/v1/decorators/gate'
+
+module Flipper
+  module Api
+    module V1
+      module Decorators
+        class Feature < SimpleDelegator
+
+          # Public: The feature being decorated.
+          alias_method :feature, :__getobj__
+          
+          # Public: Returns instance as hash that is ready to be json dumped.
+          def as_json
+            {
```

Comment:
@jnunemaker I think we should include key since:
- it will always be a string (feature#initialize  calls key = name.to_s).  That way the returned JSON value will always reflect the correct value.  If we returned name in JSON and it was a symbol as object attribute, it would serialize to JSON and be turned into a string, which it isn't since its a ruby symbol.  Could see that being confusing to a client expecting the true value.  A user can pass in a string key to flipper.feature() and it will always work since it calls to_sym

I don't think we need to include name as pretty_name like in UI because a client should just get their data and be able to make any decisions around formatting they'd like. With this change we would remove `id` from the hash.  What do you think?
```
{
  'key' => 'some_key'
  'state' => 'some_state',
  'gates' => [# some gates]
}
```